### ISSUE TITLE
Add some standard "badges" for different kinds of projects

### ISSUE THREAD

**Issue by gkthiruvathukal** [LoyolaChicagoCS/ssl#9]
Can we design some standard badges that we can put next to the different projects? One or more could appear but probably never more than a few at any given time.

- incubation - a new project that is still in early stages
- core project -  a major project within the SSL
- collaboration - a project where we are collaborating with other university/national laboratories
- documentation - a project that is focused on writing as opposed to coding
- application - a project that will lead to a full-stack application (and be hosted)
- tool - a project aimed primarily at being software tool as opposed to hosted app
- library - a project aimed at being a programming API/library
- framework - a project aimed at being a framework for app development

These badges would be similar to the ones used on GitHub, etc., e.g. for builds passing, code coverage, etc.

**Comment by morganrmarie** [comment]
I really like this idea! Would you also want to display some type of icon legend under the "Research Projects" heading? Or do you expect them to be self-explanatory?

**Comment by gkthiruvathukal** [comment]
Yes, the ideal is self-explanatory. With our nice "single page" site design, you can have a section that explains the icons. I will be happy to make even clearer descriptions of each icon, once we get the icon set created.
